src/components/manage/Blocks/DataFigure/Svg.jsx

```diff
@@ -23,18 +22,12 @@ import {
 const Svg = ({ data, detached }) => {
   const [svg, setSVG] = React.useState();
   const dispatch = useDispatch();
-  const { corsProxyPath = '/cors-proxy', host, port } = settings;
 
-  const base = __SERVER__
-    ? `http://${host}:${port}`
-    : `${window.location.protocol}//${window.location.host}`;
-
-  const path = `${base}${corsProxyPath}/${data.url}`;
 
   React.useEffect(() => {
     if (data.url.includes('.svg')) {
       if (!isInternalURL(data.url)) {
-        dispatch(getSVG(path))
+        dispatch(getProxiedExternalContent(data.url, { headers: { Accept: 'image/svg+xml', } }))
```

Comment:
Could you please tweak the volto-corsproxy to mixin the request headers? 

In theory, by default, it's really important that we don't pass the authentication headers, and I don't know if, if you overwrite the headers, if the authentication headers are not still sent.

Reply:
Done!

Reply:
https://github.com/eea/volto-corsproxy/pull/1

Reply:
You should still keep `...request` in the action in corsproxy